### PR TITLE
Add empty parentheses to method names

### PR DESCRIPTION
**Pull request by bocharsky-bw**

### CODE REVIEW

**Comment by xabbuh** [comment]
👍 

Status: Reviewed

**Comment by javiereguiluz** [comment]
👍

**Comment by javiereguiluz** [comment]
A missing one in security/api_key_authentication.rst:

```
This will provide a method ``onAuthenticationFailure`` which
```

**Comment by javiereguiluz** [comment]
Another missing one in best_practices/security.rst:

```
matches the value returned by the ``getAuthorEmail`` method
```

**Comment by javiereguiluz** [comment]
Another one in bundles/extension.rst:

```
your load method looks like
```

**Comment by javiereguiluz** [comment]
Another one in components/config/definition.rst:

```
The ``canBeDisabled`` method looks
```

I won't add more comments here ... but you get the idea that there are a lot of other cases 😁

**Comment by bocharsky-bw** [comment]
Well, regular expressions rock!

@javiereguiluz Let me know if I missed something... ;)

File: components/form.rst [hunk 1]
```diff
@@ -685,7 +685,7 @@ method to access the list of errors. It returns a
 
         $errorsAsArray = iterator_to_array($form->getErrors());
 
-    This is useful, for example, if you want to use PHP's ``array_`` function
+    This is useful, for example, if you want to use PHP's ``array_()`` function
```
Comment (javiereguiluz) [inline components/form.rst]: `array_()` -> `array_*()`

Comment (bocharsky-bw) [inline components/form.rst]: Fixed

**Comment by xabbuh** [comment]
Thank you @bocharsky-bw.

**Pull request merged**